package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "world-countries-capitals",
-	"version": "3.1.0",
+	"version": "3.2.1",
```

Comment:
Please revert that change from this _Pull Request_. It should not be a part of this feature.
(I will have a look at it later)

Reply:
Sorry but I don't think i got what's to be done here.